Article/Article.js

```diff
@@ -1,20 +1,25 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.article.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
+    // adding hide button for stretch
+    this.retractButton = this.article.querySelector('.retractButton');
+    this.retractButton.textContent = 'hide';
+    this.retractButton.addEventListener('click', () => this.expandArticle());
```

Comment:
This might be a misunderstanding of what was meant by the readme file when it said to "Add a close (or 'read') button to each Article component."

A hide button would actually be different from a close button. Hiding the article means not showing the text, but still keeping the title and expand button around. Closing it, on the other hand, means removing the article component from the DOM entirely.

That said, great idea with creating a separate hide button and implementing the `"hide-btn"` class!